cmd/nerdctl/save_linux_test.go

```diff
@@ -33,17 +33,17 @@ import (
 
 func TestSave(t *testing.T) {
 	base := testutil.NewBase(t)
-	base.Cmd("pull", testutil.AlpineImage).AssertOK()
+	base.Cmd("pull", testutil.CommonImage).AssertOK()
 	archiveTarPath := filepath.Join(t.TempDir(), "a.tar")
-	base.Cmd("save", "-o", archiveTarPath, testutil.AlpineImage).AssertOK()
+	base.Cmd("save", "-o", archiveTarPath, testutil.CommonImage).AssertOK()
 	rootfsPath := filepath.Join(t.TempDir(), "rootfs")
 	err := extractDockerArchive(archiveTarPath, rootfsPath)
 	assert.NilError(t, err)
 	etcOSReleasePath := filepath.Join(rootfsPath, "/etc/os-release")
 	etcOSReleaseBytes, err := os.ReadFile(etcOSReleasePath)
 	assert.NilError(t, err)
 	etcOSRelease := string(etcOSReleaseBytes)
-	t.Logf("read %q, extracted from %q", etcOSReleasePath, testutil.AlpineImage)
+	t.Logf("read %q, extracted from %q", etcOSReleasePath, testutil.CommonImage)
 	t.Log(etcOSRelease)
 	assert.Assert(t, strings.Contains(etcOSRelease, "Alpine"))
```

Comment:
The test is specific to Alpine, so you can retain `testutil.AlpineImage` for this test

Reply:
fixed